src/components/earlyadopter/ConnectAccount.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+  <div class="pt-20 pb-20 status-background">
+    <div class="max-w-6xl mx-auto">
+      <div class="justify-center">
+        <h1 class="font-bold text-xl">
+          Early adopter participation confirmation
+        </h1>
+        <div>
+          <div>
+            <div
+              class="
+                mt-1
+                w-full
+                rounded-md
+                bg-white
+                dark:bg-darkGray-900
+                border border-gray-300
+                dark:border-darkGray-500
+              "
+            >
+              <div
+                v-if="!showConnectMetamsk"
+                class="transform text-gray-400 dark:text-darkGray-500"
+              >
+                Loading....
+              </div>
+
+              <h3
+                v-if="showConnectMetamsk"
+                class="
+                  text-lg
+                  font-extrabold
+                  text-blue-900
+                  dark:text-white
+                  mb-6
+                  text-center
+                "
+              >
+                Choose Account that you want to apply the lockdrop bonus.
+              </h3>
+
+              <div v-if="showConnectMetamsk && filterAccounts.length === 0">
+                <h3>You did not participated in the Polkadot Crowdloan</h3>
+              </div>
+              <ul
+                v-else
+                class="
+                  max-h-56
+                  rounded-md
+                  py-1
+                  text-base
+                  overflow-auto
+                  focus:outline-none
+                "
+              >
+                <AccountOption
+                  v-for="(account, index) in filterAccounts"
+                  :key="index"
+                  :key-idx="index"
+                  :address="account"
+                  :addressName="allAccountNames[index]"
+                  :contributed="allContributedD[index]"
+                  :checked="selAccount === index"
+                  v-model:selOption="selAccount"
+                />
+              </ul>
+
+              <div class="border-2 border-dashed" />
+
+              <div class="my-2">
+                <MetamaskOption
+                  v-show="showConnectMetamsk"
+                  @connectMetamask="connectMetamask"
+                />
+                <div v-if="isResultReward" class="mt-3 font-bold">
+                  <div v-if="prevLockdropInfo">
```

Comment:
My address `0x01734005354d569716291cD1CFbc67f3f56a0b6F` participated in the first lockdrop, but the page says that I did not participate in it.
![bandicam 2021-12-22 20-10-14-098](https://user-images.githubusercontent.com/40356749/147143252-b555a53e-60a8-4fc0-a27f-18657a45d2d0.jpg)

Furthermore, if the user is eligible for the lockdrop bonus, there should be a button to apply the bonus where clicking the button will send the user's Ethereum address and Polkadot address that was used to contribute to the crowdloan. This information should be saved to the database and later exported for our reward calculation.
The early bird UI will check if the Ethereum address is already submitted or not (prevent double submission).